content/articles/generic-programming-in-c#/index.md

```diff
@@ -0,0 +1,182 @@
+
```

Comment:
When I submit my articles, I normally like adding the title and a description at the top. Consider doing this, it is pretty helpful.